tavla/app/components/actions.ts

```diff
@@ -0,0 +1,71 @@
+'use server'
+
+import { TFormFeedback, getFormFeedbackForError } from 'app/(admin)/utils'
+
+async function postForm(prevState: TFormFeedback | undefined, data: FormData) {
+    const email = data.get('email') as string
+    const message = data.get('message') as string
+
+    if (!email) {
+        return getFormFeedbackForError('auth/missing-email')
+    }
+    if (!message) {
+        return getFormFeedbackForError('contact/message-missing')
+    }
```

Comment:
denne får jeg aldri opp, får bare den engelske feilmeldingen jeg la igjen bilde av i en annen kommentar

Reply:
stemmer, det er egentlig en "lik" sjekk bare med server side validation. Har det med for vår del, siden klient side validering kan unngås (f.eks. gjennom inspekt før man trykker send).